spec/tree_spec.rb

```diff
@@ -1,14 +1,54 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
-  it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+describe AppleTree do
+  let(:tree) { AppleTree.new }
+
+  it { expect(tree.class).to be AppleTree  }
+
+  it 'will age' do
+  	tree.age!
+    expect(tree.age).to eq 1
+  end
```

Comment:
One option here is to set the age to a random number, then make it age, then check that the new age is one higher than the old one.